voice/voice-ai/xTests/AppleSignInManagerTests.swift

```diff
@@ -0,0 +1,69 @@
+import XCTest
+@testable import Voice_AI
+import UIKit
+import AuthenticationServices
+
+class MockAppleIDCredential {
+    var user: String
+    var fullName: PersonNameComponents?
+    var email: String?
+    
+    init(user: String, fullName: PersonNameComponents, email: String) {
+        self.user = user
+        self.fullName = fullName
+        self.email = email
+    }
+}
+
+class AppleSignInManagerTests: XCTestCase {
```

Comment:
should it test that after sign-in, some UI elements are displayed correctly / some actions become available on UI?